sphinxcontrib/prettyspecialmethods.py

```diff
@@ -13,9 +13,13 @@
 from docutils.nodes import Text, emphasis, inline
 from sphinx.transforms import SphinxTransform
 
-if False:
-    # For type annotations
-    from typing import Any, Dict  # noqa
+try:
+    from typing import TYPE_CHECKING
+except ImportError:
+    TYPE_CHECKING = False
```

Comment:
Typing is in 3.5 anyway?

https://docs.python.org/3.5/library/typing.html

Reply:
@eric-wieser Yeah, but [`TYPE_CHECKING` is 3.5.2](https://docs.python.org/3/library/typing.html#constant). Moot anyway with the version dependency bump. Removing it.